packages/pigeon/lib/generator_tools.dart

```diff
@@ -163,8 +165,9 @@ class Indent {
 }
 
 /// Create the generated channel name for a [func] on a [api].
-String makeChannelName(Api api, Method func) {
-  return 'dev.flutter.pigeon.${api.name}.${func.name}';
+String makeChannelName(Api api, Method func, String? packageName) {
```

Comment:
What are the circumstances where this would be null?

Reply:
Shouldn't happen.